src/composables/useBalanceCalculator.js

```diff
@@ -0,0 +1,91 @@
+import { tableAttribut } from './useLookUps.js';
+
+let bilanz = [];
+let errors = [];
+let calcCultures = [];
+
+/**
+ * returns Array
+ */
+function calculateBilanz() {
+  console.log('calc', calcCultures);
+  return [];
+}
+
+/**
+ * returns Array
+ */
+export function updateBilanz(entry) {
```

Comment:
@ahocevar - kannst du mir zeigen, wie da das korrekte Type Scripting wäre?

Reply:
Ganz grundsätzlich ist die Syntax `@returns {Type}`, nicht `@returns Type`. Und `Array` ist halt sehr generisch. Ich mach gute Types in einen separaten Commit, dann siehst du, wie man's machen kann.

Falls nicht bekannt, hier ist die Referenz: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html